repo2docker/buildpacks/r.py

```diff
@@ -180,7 +195,34 @@ def get_build_scripts(self):
             self.checkpoint_date.isoformat()
         )
 
-        scripts = [
+        scripts = []
+        if self.r_version == "3.6":
```

Comment:
Maybe I am being silly, but what happens if other R versions are given? I couldn't find a spot where a different R installation command is triggered if the specified version isn't 3.6

Reply:
oops, I just read through your first comment again, I see there's no 3.5 support yet but that's on the horizon, so feel free to ignore this :-)

Reply:
Nothing yet because this PR doesn't even work properly if you don't specify 3.6 :)